modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ConfigurationGenerator.java

```diff
@@ -146,9 +148,11 @@
      */
     public static final String PARAM_TEMPLATES_NAME = "nuxeo.templates";
 
+    @Deprecated(forRemoval = true) // not used
```

Comment:
Can you add `since  = "11.5"` too in all the new `@Deprecated`?

Reply:
These were added to help me during the refactoring, I will clean intermediate commits and remove them.